.github/workflows/test.yml

```diff
@@ -7,27 +7,9 @@ on:
     branches:
     - "**"
   pull_request:
+  schedule:
+    - cron: '0 0 * * *'
```

Comment:
Why CI at 12am?

Reply:
I want this to run once a day every single day. This way, if we upgrade the linter to v1.50.0 but don't update this repo in 6 months, we'll know that it's broken.

Reply:
Sounds good.